iconservice/icon_service_engine.py

```diff
@@ -517,9 +518,12 @@ def _process_issue_transaction(self,
     def _invoke_issue_request(self,
                               context: 'IconScoreContext',
                               request: dict) -> 'TransactionResult':
-        issue_data_in_tx = request['params']['data']
+        if not isinstance(request['params']['data'], dict):
```

Comment:
I want you add key error assert before this code.

assert 'params' in request
assert 'data' in request['params']